### PR TITLE
Add storage tests

### PR DESCRIPTION
**Pull request by wardetu**

### CODE REVIEW

File: src/test/data/JsonUserPrefsStorageTest/ExtraValuesUserPref.json [hunk 1]
```diff
@@ -9,5 +9,5 @@
       "z" : 99
     }
   },
-  "addressBookFilePath" : "addressbook.json"
+  "addressBookFilePath" : "resume.json"
```
Comment (helloImHai) [inline src/test/data/JsonUserPrefsStorageTest/ExtraValuesUserPref.json:12]: is it possible to change the addressBookFilePath to resumeBookFilePath or smt?

File: src/test/java/seedu/address/storage/JsonSerializableResumeBookTest.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+        // Equals
+        JsonSerializableResumeBook dataFromFile = JsonUtil.readJsonFile(TYPICAL_RESUME_BOOK_FILE,
+                JsonSerializableResumeBook.class).get();
+
+        // Not equals
+        JsonSerializableResumeBook alternateDataFromFile = JsonUtil.readJsonFile(OTHER_TYPICAL_RESUME_BOOK_FILE,
+                JsonSerializableResumeBook.class).get();
+
+        assertEquals(dataFromFile, new JsonSerializableResumeBook(typicalResumeBook));
```
Comment (helloImHai) [inline src/test/java/seedu/address/storage/JsonSerializableResumeBookTest.java:165]: Maybe bring this up under // Equals and above // Not equals?

File: src/main/java/seedu/address/model/item/Skill.java [hunk 3]
```diff
@@ -64,6 +64,7 @@ public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Skill // instanceof handles nulls
                 && getName().equals(((Skill) other).getName())
-                && level.getLevelCode() == ((Skill) other).level.getLevelCode());
+                && level.getLevelCode() == ((Skill) other).level.getLevelCode()
+                && tags.containsAll(((Skill) other).tags));
```
Comment (helloImHai) [inline src/main/java/seedu/address/model/item/Skill.java]: ((Skill) other).tags.containsAll(tags) too

**Review by helloImHai — CHANGES_REQUESTED** [review]
As it should be m8 💯💯💯
Few very minor comments

File: src/test/data/JsonUserPrefsStorageTest/ExtraValuesUserPref.json (hunk 1 above)
Comment (wardetu) [reply src/test/data/JsonUserPrefsStorageTest/ExtraValuesUserPref.json:12]: Yes I will make that change

**Pull request merged**